docs/getting-started/installing-data.md

```diff
@@ -38,4 +38,22 @@ This should output a version number, for example on my machine it shows:
 ```
 0.4.1
 ```
+\- To get all the available commands:
+```
+data help
+```
+
+\- Get info on data (file or dataset) at path or url
```

Comment:
I think this sentence is incorrect..

Reply:
@anuveyatsu then we have to fix main help message, it was taken from there

Reply:
@Mikanebu sorry, just re-read it and it seems OK. Please, just remove all unnecessary examples and leave only info command.